pysatNASA/instruments/timed_guvi.py

```diff
@@ -0,0 +1,351 @@
+# -*- coding: utf-8 -*-
+"""Supports the GUVI instrument on TIMED.
+
+Downloads data from the NASA Coordinated Data
+Analysis Web (CDAWeb).
+
+Supports two options for loading that may be
+specified at instantiation.
+
+Properties
+----------
+platform
+    'timed'
+name
+    'guvi'
+tag
+    'low'
+    'high'
+inst_id
+    'imaging'
+    'spectrograph'
+
+Note
+----
+
+Warnings
+--------
+Currently no cleaning routine.
+
+Example
+-------
+::
+
+    import pysat
+    guvi = pysat.Instrument(platform='timed', name='guvi', inst_id='imaging', tag='low')
+    guvi.download(dt.datetime(2005, 6, 28), dt.datetime(2005, 6, 29))
+    guvi.load(yr=2005, doy=171)
```

Comment:
```suggestion
    guvi.load(date=dt.datetime(2005, 6, 28))
```
The downloaded data is for day 179.  EIther update for consistency or use datetime in the example.